templates/templates/preview/clientconfig_plain_text_info.template

```diff
@@ -0,0 +1,24 @@
+
+        Client Configuration:
+          {%- for db in manifest.dbs %}
+          {%- if db.name %}
+          Database Identifier: {{ db.name }}
+        {%- endif %}
+            Database Backend: {{ config.database_cli | get_dbtype }}
+            TCP/IP port:  
+            {%- for port in db.ports %}
+              {{ port }}
+            {%- endfor %}
+            {%- if db.num_shards > 2 %}
+            Type: Clustered
+            {%- else %}
+            Type: Standalone
+            {%- endif %}
+        {%- endfor %}
+          {%- if model.query_key_prefixing() %} 
+          Outgoing Key Collision Prevention (Key Prefixing): 
+            Tensors: {{ model.query_key_prefixing() | as_toggle }}
+            Datasets: {{ model.query_key_prefixing() | as_toggle }}
+            ML Models/Torch Scripts: {{ False | as_toggle }}
+            Aggregation Lists: {{ model.query_key_prefixing() | as_toggle }}
+        {%- endif %}
```

Comment:
I really want you to try out `pre-commit` and just start w/a configuration that adds trailing newlines... it will be nice when you never have to hear me ask, won't it!?

```
repos:
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.5.0
    hooks:
    -   id: check-yaml
    -   id: end-of-file-fixer
    -   id: trailing-whitespace
```

Reply:
woops! yep I will add